lab_3/main.py

```diff
@@ -119,15 +228,41 @@ def _calculate_distance(self, first_n_grams: tuple, second_n_grams: tuple) -> in
         :param second_n_grams: a tuple of the top_k n-grams
         :return: a distance
         """
-        pass
+        if not isinstance(first_n_grams, tuple) or not isinstance(second_n_grams, tuple) or \
+                None in first_n_grams or None in second_n_grams:
```

Comment:
`None in first_n_grams or None in second_n_grams` - too specific, you should try to check that passed arguments are _appropriate_.